src/main/java/backend/model/event/Team.kt

```diff
@@ -193,7 +193,7 @@ class Team : BasicEntity, Blockable {
     }
 
     private fun Sponsoring.toEmailListing(): String {
-        return "<b>Name</b> ${this.sponsor.firstname} ${this.sponsor.lastname} <b>Status</b> ${this.status} <b>Betrag pro km</b> ${this.amountPerKm.display()} <b>Limit</b> ${this.limit.display()} <b>Gereiste KM</b> ${this.team?.getCurrentDistance()} <b>Spendenversprechen</b> ${this.billableAmount().display()}"
+        return "<b>Name</b> ${this.sponsor?.firstname} ${this.sponsor?.lastname} <b>Status</b> ${this.status} <b>Betrag pro km</b> ${this.amountPerKm.display()} <b>Limit</b> ${this.limit.display()} <b>Gereiste KM</b> ${this.team?.getCurrentDistance()} <b>Spendenversprechen</b> ${this.billableAmount().display()}"
```

Comment:
will the string default to `""` or `null`?

Reply:
`""` would be fine for now, `null` would be weird in emails, if it accidentally happens, e.g. with donation reminders after an account was deleted already

Reply:
I will make an issue to handle those